supply/lib/supply.rb

```diff
@@ -23,6 +24,13 @@ class << self
 
   CHANGELOGS_FOLDER_NAME = "changelogs"
 
+  # https://developers.google.com/android-publisher/#publishing
+  AVAILABLE_TRACKS = %w(production rollout beta alpha internal)
+  DEFAULT_TRACK = AVAILABLE_TRACKS[0]
+
+  # https://developers.google.com/android-publisher/api-ref/edits/tracks
+  RELEASE_STATUS = %w(completed draft halted inProgress)
```

Comment:
Move these statuses also to constants so we can use them other places without hardcoding strings